rosbag2_compression/test/rosbag2_compression/test_sequential_compression_writer.cpp

```diff
@@ -243,7 +243,7 @@ TEST_F(SequentialCompressionWriterTest, writer_creates_correct_metadata_relative
 
   tmp_dir_storage_options_.max_bagfile_size = 1;
   writer_->open(tmp_dir_storage_options_);
-  writer_->create_topic({test_topic_name, test_topic_type, "", ""});
+  writer_->create_topic({test_topic_name, test_topic_type, "", ""}, {});
```

Comment:
@james-rms I would recommend to revert to use original API without message definition in tests to reduce amount of unrelated delta.
This is also relates to all other occurrences of usage create_topic(..) with empty message_definition in this and all other files with old unit tests.